qtconsole/console_widget.py

```diff
@@ -2089,7 +2088,9 @@ def _insert_plain_text(self, cursor, text, flush=False):
         if self._executing and end_doc_pos - end_scroll_pos <= 1:
             end_scroll = (self._control.verticalScrollBar().maximum()
                           - self._control.verticalScrollBar().pageStep())
-            self._control.verticalScrollBar().setValue(end_scroll)
+            # only go up
```

Comment:
don't you mean scoll *down*?

Reply:
Well the text goes up

Reply:
But I can change the comment

Reply:
I find `# Only scroll down` easier to understand. Sorry for the nitpick ;)